xarray/tests/test_backends.py

```diff
@@ -3397,7 +3395,7 @@ def test_region_write(self) -> None:
                     "set": 5,
                     "get": 10,
                     "list_dir": 3,
-                    "list_prefix": 0,
+                    "list_prefix": 4,
```

Comment:
wild. zarr is calling `list_prefix(prefix='')` on group open; and then listing each variable:

```
listing ''
listing ''
listing 'x/'
listing 'foo/'
```

cc @d-v-b